packages/maumau-server/src/controllers/pool.ts

```diff
@@ -44,9 +45,19 @@ export default class PoolController {
     );
   }
 
-  private join = (request: Request<{}, {}, { id: string; name: string }>, response: Response<{ status: Status }>) => {
-    const { id, name } = request.body;
-    this.matchmakerService.joinPool({ id, name });
+  private join = (
+    request: Request<{}, {}, { id: string; name: string; playAgainstBot: boolean | undefined }>,
+    response: Response<{ status: Status }>,
+  ) => {
+    const { id, name, playAgainstBot } = request.body;
+
+    // Compatibilty with old clients:
```

Comment:
Wir können oben in der Validierung einen default anlegen. Das ist einfacher denke ich:
```
body: Joi.object({
          id: Joi.string().uuid().required(),	       
          name: Joi.string().required(),	       
          playAgainstBot: Joi.boolean().default(true), <-----
        }),	        
}),
```

Reply:
Ah nice, werd ich so machen.